.github/workflows/maven.yml

```diff
@@ -5,6 +5,9 @@ name: Maven unit tests
 
 on:
   push:
+    branches:
+      - master
+      - /.*-stable/
```

Comment:
I saw this in `prometheus-operator` Github actions. We don't use the `*-stable` pattern, is that something you want to introduce?